aframe/pipelines/sandbox/sandbox.cfg

```diff
@@ -27,6 +27,7 @@ waveform_approximant = IMRPhenomPv2
 waveform_duration = 8
 minimum_frequency = 20
 reference_frequency = 50
+coalescence_time = 6
```

Comment:
Just as a clarification, with `waveform_duration` of 8 this means the coalescence time is 2 seconds from the right edge?

Reply:
That's correct. I chose this time for testing semi-arbitrarily, I just wanted something that wasn't in the center.